src/style.css

```diff
@@ -1,3 +1,101 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap';
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --nav-height: 60px;
+  --margin-between-items: 20px;
+  --blue-color: #00ACDC;
+  --underline-width: 36px;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.logo {
+  display: flex;
+  width: 40px;
+  margin-left: 50px;
+  flex-shrink: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: var(--nav-height);
+}
+
+.navbar {
+  margin-right: 50px;
+}
+
+.nav__item {
+  margin-right: var(--margin-between-items);
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: block;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding instead

Reply:
![image](https://github.com/mate-academy/layout_moyo-header/assets/115790814/287b675e-92d7-4dc4-bf88-fd1fc77caf84)